tests/unit/python/.coveragerc

```diff
@@ -0,0 +1,16 @@
+[run]
+omit =
+    # Ignore files
+    */__init__.py
+    */__template__.py
+    */setup.py
+    */python/fledge/tasks/north/sending_process.py
+    # Omit directory
+    */python/fledge/plugins/common/*
+    */python/fledge/plugins/filter/*
+    */python/fledge/plugins/north/*
+    */python/fledge/plugins/south/*
+    */python/fledge/plugins/notificationDelivery/*
+    */python/fledge/plugins/notificationRule/*
```

Comment:
No such directories exist.

Reply:
correct, these are only once we installed python based notification plugins. This is only for local setup if anyone wants to check coverage report.